graphql-java-servlet/src/main/java/graphql/kickstart/servlet/HttpRequestHandlerImpl.java

```diff
@@ -24,28 +23,30 @@ public HttpRequestHandlerImpl(GraphQLConfiguration configuration) {
             new QueryResponseWriterFactoryImpl()));
   }
 
-  public HttpRequestHandlerImpl(
+  HttpRequestHandlerImpl(
       GraphQLConfiguration configuration, HttpRequestInvoker requestInvoker) {
     this.configuration = configuration;
     this.requestInvoker = requestInvoker;
   }
 
   @Override
   public void handle(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    if (request.getCharacterEncoding() == null) {
+      request.setCharacterEncoding(StandardCharsets.UTF_8.name());
+    }
+
+    ListenerHandler listenerHandler =
+        ListenerHandler.start(request, response, configuration.getListeners());
```

Comment:
Make sure to apply proper google java format which among other things uses two spaces instead of four.